raiden-ts/src/raiden.ts

```diff
@@ -955,6 +956,29 @@ export class Raiden {
       )
       .toPromise();
   }
+
+  /**
+   * Mints the amount of tokens of the provided token address.
+   * Throws an error if executed on main net.
+   *
+   * @param tokenAddress - Address of the token to be minted
+   * @param amount - Amount to be minted
+   * @returns transaction
+   */
+  public async mint(tokenAddress: string, amount: BigNumberish): Promise<ContractTransaction> {
```

Comment:
Please, avoid forcing dependency of ethers specific type here. For consistency, other transactioning methods return `Hash`, which is assignable to normal `string` containing txHash. Also for consistency, the parameter is usually called just `token`, as public interface only deals with addresses anyway.
```suggestion
  public async mint(token: string, amount: BigNumberish): Promise<Hash> {
```